### ISSUE TITLE
Some telescopes have no clock files in PINT

### ISSUE THREAD

**Issue by aarchiba** [nanograv/PINT#1235]
You cannot even load a `.tim` file with (say) `meerkat` data in it unless you have a `TEMPO2` environment variable set. Should we import the clock files from TEMPO and TEMPO2?

```python
for a,o in pint.observatory.Observatory._registry.items():
    if hasattr(o, "clock_dir") and o.clock_dir != "PINT":
        print(a, o.clock_dir)
```
gives
```
meerkat TEMPO2
parkes TEMPO2
nancay TEMPO2
ncyobs TEMPO2
effelsberg TEMPO2
effelsberg_pre_2021 TEMPO2
gmrt TEMPO2
wsrt TEMPO2
most TEMPO2
```

(Incidentally is there a way to list PINT's known observatories without poking inside an underscore variable?)

**Comment by aarchiba** [comment]
This is a narrower, temporary fix fthan #1180

**Comment by aarchiba** [comment]
In light of #1239 and #1247 I think this is under control.
